From e141fcf509a035473a289c77a9f4ead7268787ea Mon Sep 17 00:00:00 2001
From: David Wittman <dwittman@gmail.com>
Date: Sat, 14 Jan 2017 17:08:23 -0600
Subject: [PATCH] staging: ks7010: Fix brace style issue in ks_wlan_net.c

This change fixes a checkpatch error for "that open brace { should be
on the previous line" as well as a related spacing warning.

Signed-off-by: David Wittman <dwittman@gmail.com>
---
 drivers/staging/ks7010/ks_wlan_net.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/ks7010/ks_wlan_net.c b/drivers/staging/ks7010/ks_wlan_net.c
index e5d04ad..8c55428 100644
--- a/drivers/staging/ks7010/ks_wlan_net.c
+++ b/drivers/staging/ks7010/ks_wlan_net.c
@@ -3400,8 +3400,9 @@ int ks_wlan_close(struct net_device *dev)
 /* Operational parameters that usually are not changed. */
 /* Time in jiffies before concluding the transmitter is hung. */
 #define TX_TIMEOUT  (3*HZ)
-static const unsigned char dummy_addr[] =
-    { 0x00, 0x0b, 0xe3, 0x00, 0x00, 0x00 };
+static const unsigned char dummy_addr[] = {
+	0x00, 0x0b, 0xe3, 0x00, 0x00, 0x00
+};
 
 static const struct net_device_ops ks_wlan_netdev_ops = {
 	.ndo_start_xmit = ks_wlan_start_xmit,
-- 
1.9.1

